example/app/AppComponents.scala

```diff
@@ -17,11 +17,13 @@ class AppComponents(context: ApplicationLoader.Context)
   with AhcWSComponents
   with HttpFiltersComponents{
 
-  val clientId: String = configuration.get[String]("your.clientId.config.path")
-  val clientSecret: String = configuration.get[String]("your.clientSecret.config.path")
-  val redirectUrl: String = configuration.get[String]("your.redirectUrl.config.path")
-  val domain: String = configuration.get[String]("your.apps-domain.config.path")
-  val googleAuthConfig = GoogleAuthConfig(clientId, clientSecret, redirectUrl, domain)
+  val googleAuthConfig = GoogleAuthConfig(
+    clientId = configuration.get[String]("your.clientId.config.path"),
+    clientSecret = configuration.get[String]("your.clientSecret.config.path"),
+    redirectUrl = configuration.get[String]("your.redirectUrl.config.path"),
+    domain = configuration.get[String]("your.apps-domain.config.path"),
+    antiForgeryChecker = AntiForgeryChecker.borrowSettingsFromPlay(httpConfiguration)
```

Comment:
Here we're using the new `borrowSettingsFromPlay()` convenience method, borrowing the settings Play uses to sign the JWT-encoded `PLAY_SESSION` cookie - we're saying, if Play trusts this encoding and algorithm for signing it's session cookie, then we also trust them for signing our anti-forgery `state` in the OAuth transaction.

Play currently defaults to using [`HMAC-SHA-256`](https://github.com/wsargent/playframework/blob/e96c39441239ce30c5e0742c70ea2871e24edd33/framework/src/play/src/main/resources/reference.conf#L158) as the Json Web token signing algorithm.